spec/features/saml/saml_spec.rb

```diff
@@ -199,6 +199,8 @@
         click_on 'Yes'
 
         complete_idv_profile_ok(user.reload)
+        click_submit_default
+        enter_correct_otp_code_for_user(user)
```

Comment:
We can certainly test those 2 cases -- this existing SAML spec was making assumptions about the phone number being the same. It was the un-specified assumption that caused the test to fail.

Should we add a second feature test with use case (1) as well?